src/coreComponents/mesh/FaceElementSubRegion.cpp

```diff
@@ -127,37 +133,50 @@ void FaceElementSubRegion::copyFromCellBlock( FaceBlockABC const & faceBlock )
 
     auto const it = std::max_element( s.cbegin(), s.cend() );
     integer const maxSize = *it;
-    m_elementType = f( maxSize );
+    m_elementType = hack( maxSize );
     m_numNodesPerElement = maxSize;
   }
 
-  // The `m_surfaceElementsToCells` mappings involves element, sub regions and regions indices.
+  // The `m_2dElemToElems` mappings involves element, sub regions and regions indices.
   // We store the element indices that are correct.
   // But we only have access to the cell block indices, not the sub regions indices.
   // Temporarily, and also because they share the same dimensions,
   // we store the cell block mapping at the sub region mapping location.
-  // It will later be transformed into a sub regions mapping.
-  // Last, we fill the regions mapping with dummy -1 values that should all be replaced eventually.
+  // It will later be transformed into a sub regions mapping.  // Last, we fill the regions mapping with dummy -1 values that should all be replaced eventually.
```

Comment:
```suggestion
  // It will later be transformed into a sub regions mapping.  
  // Last, we fill the regions mapping with dummy -1 values that should all be replaced eventually.
```